tests/test_cli.py

```diff
@@ -1,6 +1,9 @@
+import json
+
 import pytest
 
 from tests.testing_utils.fixtures import *
+from tests.testing_utils.result_assertions import find_conflicts_in_options_dict
```

Comment:
I would keep it here, in this file. This util won't be uses probably anywhere else than this tests, right?

Reply:
Done